codespell_lib/data/dictionary.txt

```diff
@@ -25278,8 +25281,11 @@ stoppping->stopping
 stopps->stops
 stopry->story
 storag->storage
+storeable->storable
 storeage->storage
 stoream->stream
+storeble->storable
+storeing->storing
 storeis->stories
```

Comment:
I guess you could also add a suggestion of `store is` here?

Reply:
I'm not sure if it is wanted to split up words like that. But if it is, I will add more of those combinations.

Reply:
Maybe comments/consent of @peternewman

Reply:
and perhaps storeis -> stores is also a reasonable suggestion? :shrug:

Reply:
added

Reply:
Also storeys (again).

> I'm not sure if it is wanted to split up words like that. But if it is, I will add more of those combinations.
> Maybe comments/consent of @peternewman

You don't ever need my consent. Worse case if you add it, myself or someone else reviewing may suggest it gets removed if it seems unlikely.

I'd say generally, if it's two valid words, then the typo may be they missed the space bar (rather than another letter), so go for it. If you want to be really keen, try a search for that pair adjacent. I guess there might be some pairs that are unlikely to occur together, but I'm struggling to think of good examples (e.g. book door; perhaps the hypothetical typo meant backdoor, although even then "I want to book door to door service").

Reply:
I wanted to ask, before i search for all typos like that.

Reply:
Okay, do you want to add the two specific fixes for this into here so it can be merged, then do another PR for the other stuff.

It might even be worth writing some sort of script that splits each word in the dictionary in two at every possible boundary and then checks each resulting word against aspell. Or just start with a list of common 1-3 letter words (like a, in, is, it) and see if they are prefixes or suffixes of the dictionary words (and check the resulting word is valid too).

Reply:
I would put all the split up in a separate PR. It makes it easier to review and revert if necessary.

I'm probably going to make it semi-automatic.

Reply:
> I would put all the split up in a separate PR. It makes it easier to review and revert if necessary.

Can you fix this one now we've found it.

I'm saying don't drop a PR with 300 changes in one PR split it up into chunks, as otherwise it ends up like #1624 that becomes a beast that's a challenge to get over the line.

> I'm probably going to make it semi-automatic.

Good, makes sense.

Reply:
I was misleading. I meant "I want to put this in (one of the multiple) new MR(s)

added the suggestion.